tests/test_driver.py

```diff
@@ -19,40 +15,33 @@
 
 class TestRelayDriver(unittest.TestCase):
     def test_constructor(self):
-        GPIO_mock.setmode.assert_called_with("MockedBCM")
-        relay = Relay(18, True)
+        relay = Relay.get_or_create_relay(1, True, MockFactory())
         self.assertIsInstance(relay, Relay)
-        self.assertEqual(relay.pin, 18)
+        self.assertEqual(relay.pin, 1)
```

Comment:
```suggestion
        self.assertEqual(relay.pin, 18)
```

please preserve the pin numbers of the tests, only change logic, not data (here and below)

Reply:
fixed